src/utils/index.js

```diff
@@ -51,12 +51,23 @@ function listExtensionsPaths () {
     path.resolve(projectRoot, extPath)
   )
 }
+function validateEntryPointIncludedInPackage (entryPointPath) {
+  const entryPointPaths = listExtensionsPaths()
+  if (
+    !entryPointPaths.includes(path.resolve(entryPointPath))
+  ) {
+    throw new Error(
+      `O caminho especificado (${entryPointPath}) não foi declarado como de uma extensão no package.json em quoti.extensions`
+    )
```

Comment:
@luizAlbuquerqueJr, acho que a mensagem fica melhor assim, o usuário não precisa, necessariamente, saber que as extensões que ele selecionou estão em `quoti.extensions`. Com essa mensagem, ele vai verificar o path ou vai fazer select-extension. O que tu acha?
```suggestion
    throw new Error(
      `O entrypoint especificado (${entryPointPath}) não está entre as extensões que já foram selecionadas. Tem certeza que o caminho está correto ou que a extensão já foi selecionada com qt select-extension?`
    )
```

Reply:
done